src/views/single/canceled-status.php

```diff
@@ -11,35 +11,43 @@
  *
  * @link {INSERT_ARTICLE_LINK_HERE}
  *
- * @version 4.9.9
+ * @version TBD
  *
- * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ * @var WP_Post $event  The event post object with properties added by the `tribe_get_event` function.
+ * @var string  $status The event status.
  *
  * @see tribe_get_event() For the format of the event object.
  */
 use Tribe__Date_Utils as Dates;
 use WP_Post;
 
-$status          = get_post_meta( $event->ID, Event_Meta::$key_status, true );
-$canceled_reason = get_post_meta( $event->ID, Event_Meta::$key_status_canceled_reason, true );
-
 // Don't print anything when status for this event is not
 if ( 'canceled' !== $status ) {
 	return;
 }
 
+$canceled_reason = get_post_meta( $event->ID, Event_Meta::$key_status_canceled_reason, true );
+
+if ( $canceled_reason ) {
+	$label = _x( 'Canceled', 'Text to display canceled on event single if there is a canceled reason.', 'tribe-ext-events-control' );
+} else {
+	$label = sprintf( _x( 'This %s has been canceled', 'Text to display canceled on event single if no canceled reason.', 'tribe-ext-events-control' ), tribe_get_event_label_singular() );
```

Comment:
I was thinking we might not want to include the %s in the translation, but then we have only `This` be a translated string, which makes it harder to translate too. I think we can stick with this.